modules/tensor_mechanics/src/materials/ComputeReturnMappingStress.C

```diff
@@ -47,13 +54,22 @@ ComputeReturnMappingStress::ComputeReturnMappingStress(const InputParameters & p
     _relative_tolerance(parameters.get<Real>("relative_tolerance")),
     _absolute_tolerance(parameters.get<Real>("absolute_tolerance")),
     _output_iteration_info(getParam<bool>("output_iteration_info")),
+    _perform_finite_strain_rotations(getParam<bool>("perform_finite_strain_rotations")),
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_base_name + "elasticity_tensor")),
     _elastic_strain_old(declarePropertyOld<RankTwoTensor>(_base_name + "elastic_strain")),
-    _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment"))
+    _strain_increment(getMaterialProperty<RankTwoTensor>(_base_name + "strain_increment")),
+    _plastic_strain(declareProperty<RankTwoTensor>(_base_name + "plastic_strain")),
```

Comment:
OK, i'll rename to inelastic_strain.  Then the IsotropicPlasticityStressUpdate and HyperbolicViscoplasticityStressUpdate will not be able to use "inelastic_strain", which they currently have.  I'll change theirs to "plastic_strain" for now.